client/src/containers/add-building/FormFragment.js

```diff
@@ -187,6 +192,11 @@ const FormFragment = ({ form }) => {
           </Select>,
         )}
       </Form.Item>
+      <Form.Item label={<Trans>Captcha</Trans>}>
+        {getFieldDecorator('captcha', {
+          rules: [{ required: true, message: <EmptyFieldMessage /> }],
+        })(<HCaptcha sitekey="your-sitekey" onVerify={handleVerifyCaptcha} />)}
```

Comment:
We will need to pass the key the same way it's done for the `MAP_API_KEY` (`REACT_APP_API_KEY` in `.env.dist`)